tests/test_sourmash.py

```diff
@@ -4661,6 +4661,35 @@ def test_gather_output_unassigned_with_abundance(runtmp, prefetch_gather, linear
             assert nomatch_mh.hashes[hashval] == abund
 
 
+def test_multigather_output_unassigned_with_abundance(runtmp):
+    c = runtmp
+    query = utils.get_test_data('gather-abund/reads-s10x10-s11.sig')
+    against = utils.get_test_data('gather-abund/genome-s10.fa.gz.sig')
+
+    with pytest.raises(SourmashCommandFailed):
+        c.run_sourmash('multigather', query, against, '--output-unassigned',
+                c.output('unassigned.sig'))
+
+    assert os.path.exists(c.output('unassigned.sig'))
```

Comment:
This assert fails because multigather fails to produce any output files - which is after `c.run_sourmash(...)` raises `SourmashCommandFailed`. I'm confused as to why you would expect a failed sourmash command to produce the correct output files - those two expectations directly conflict.

Reply:
Hmm, I'll take a look at that.